windows/sound.cpp

```diff
@@ -154,15 +155,10 @@ QString CSound::CheckDeviceCapabilities()
     if ( ( SetSaRateReturn == ASE_NoClock ) || ( SetSaRateReturn == ASE_InvalidMode ) || ( SetSaRateReturn == ASE_NotPresent ) )
     {
         // return error string
-        return tr ( "The audio device does not support setting the required sampling "
-                    "rate. This error can happen if you have an audio interface like the "
-                    "Roland UA-25EX where you set the sample rate with a hardware switch "
-                    "on the audio device. If this is the case, please change the sample rate "
-                    "to " ) +
-               QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) +
-               tr ( " Hz on the "
-                    "device and restart the " ) +
-               APP_NAME + tr ( " software." );
+        return QString ( tr ( "The current audio device configuration is incompatible "
+                              "because the sample rate couldn't be set to %1 Hz. Please check for a hardware switch or "
+                              "driver setting to set the sample rate manually and restart the app." ) )

```

Comment:
What does "the app" mean here? If it's Jamulus, we should say that.

Reply:
Done.